composer.json

```diff
@@ -4,8 +4,10 @@
     "type": "psalm-plugin",
     "require": {
         "barryvdh/laravel-ide-helper": "^2.6",
+        "laravel/framework": "^5.5 || ^6.0",
```

Comment:
I was surprised that `laravel/framework` wasn't an explicit dependency, considering this is a plugin for it. How do we feel about these version constraints?

Reply:
Ah, I see this package also supports Lumen. I guess we should switch to requiring `illuminate/*` specific packages?

Reply:
I'm going to figure out these deps in a different pr